configs/config-combinations/readme.md

```diff
@@ -0,0 +1,19 @@
+This directory containers YAML files which are config combinations for various
+RBE toolchain containers.
+A config combination file indicates the latest bazel version and toolchain 
+container the corresponding toolchain configs were last generated for. When 
+newer bazel versions or toolchain containers are available, the config 
```

Comment:
When a new Bazel version or container version is available,

Reply:
Done